src/ov_utils.cpp

```diff
@@ -89,4 +89,21 @@ std::string getNetworkInputsInfoString(const InferenceEngine::InputsDataMap& inp
     }
     return stringStream.str();
 }
+
+std::string getTensorMapString(const std::map<std::string, std::shared_ptr<TensorInfo>>& inputsInfo) {
+    std::stringstream stringStream;
+    for (const auto& pair : inputsInfo) {
+        const auto& name = pair.first;
+        auto inputInfo = pair.second;
```

Comment:
Is it possible we get invalid pointer here?

Reply:
No